<?xml version="1.0" encoding="UTF-8"?>
<bajadoc version="2.0" createdBy="niagara-baja-doclet-1.0.9" createdAt="03-May-2024" createdOn="883e7f7a9875">
<class module="baja" runtimeProfile="rt" qualifiedName="javax.baja.sys.AlreadyParentedException" name="AlreadyParentedException" packageName="javax.baja.sys" public="true" category="exception">
<description>
The AlreadyParentedException is thrown when&#xa; an attempt is made to set or add a BComplex&#xa; value which is already contained by another&#xa; object.
</description>
<tag name="@author">Brian Frank</tag>
<tag name="@creation">28 Sept 00</tag>
<tag name="@version">$Revision: 2$ $Date: 2/28/01 9:44:54 AM EST$</tag>
<tag name="@since">Baja 1.0</tag>
<extends>
<type class="javax.baja.sys.BajaRuntimeException"/>
</extends>
<!-- javax.baja.sys.AlreadyParentedException(java.lang.String) -->
<constructor name="AlreadyParentedException" public="true">
<parameter name="msg">
<type class="java.lang.String"/>
</parameter>
<description>
Constructor with specified detailed message.
</description>
</constructor>

<!-- javax.baja.sys.AlreadyParentedException() -->
<constructor name="AlreadyParentedException" public="true">
<description>
Constructor with no message.
</description>
</constructor>

</class>
</bajadoc>
